oak-jcr/src/main/java/org/apache/jackrabbit/oak/jcr/session/NodeImpl.java

```diff
@@ -1679,6 +1679,24 @@ public PropertyImpl performNullable() throws RepositoryException {
         });
     }
 
+    @Override
+    public Node getParentOrNull() throws RepositoryException {
+        return perform(new NodeOperation<Node>(dlg, "getParentOrNull") {
```

Comment:
this `perform` is [marked](https://github.com/apache/jackrabbit-oak/blob/dc3b9486251bd6aa965165d2e7a02081875d7654/oak-jcr/src/main/java/org/apache/jackrabbit/oak/jcr/session/ItemImpl.java#L111) with `@NotNull` - should this use something similar to what `getNodeOrNull` uses, eg `sessionDelegate.performNullable`?

Reply:
@stefan-egli , i agree. it should call performNullable such as it's done in `Node.getNodeOrNull`.

Reply:
Done